docs/planning/backend/app/services/fragrance_service.py

```
@@ -0,0 +1,290 @@
+            launch_year=data.launch_year,
+            country=data.country,
+            primary_family=data.primary_family,
+            subfamily=data.subfamily,
+            longevity=data.longevity,
+            sillage=data.sillage,
+            rating=data.rating,
+            image_url=data.image_url,
+            fragrantica_url=data.fragrantica_url,
+            accords=data.accords or {},
+            data_source=data.data_source,
+            data_source_updated_at=datetime.utcnow(),
```

Ruff (0.14.10)

`datetime.datetime.utcnow()` used

(DTZ003)